libs/src/services/creatorNode/CreatorNodeSelection.test.js

```diff
@@ -26,42 +26,44 @@ const mockEthContracts = (urls, currrentVersion, previousVersions = null) => ({
   }
 })
 
+// Add fields as necessary
+let defaultHealthCheckData = {
+  'service': CREATOR_NODE_SERVICE_NAME,
+  'version': '1.2.3',
+  'healthy': true,
+  'country': 'US',
+  'latitude': '37.7749',
+  'longitude': '-122.4194',
+  'databaseConnections': 5,
+  'totalMemory': 6237552640,
+  'usedMemory': 6111436800,
+  'storagePathSize': 62725623808,
+  'storagePathUsed': 14723018752,
+  'maxFileDescriptors': 524288,
+  'allocatedFileDescriptors': 2912,
+  'receivedBytesPerSec': null,
```

Comment:
let's put a value for this guy. generally it should be non null, but i think depending on what's going on in local setup it might not have a value. on prod it should with uploads coming in